vmdb/app/models/vmdb_database_setting.rb

```diff
@@ -1,90 +1,41 @@
-class VmdbDatabaseSetting < ActsAsArModel
-  set_columns_hash(
-    :name              => :string,
-    :description       => :string,
-    :value             => :string,
-    :minimum_value     => :integer,
-    :maximum_value     => :integer,
-    :unit              => :string,
-    :vmdb_database_id  => :integer
-  )
+class VmdbDatabaseSetting < ActiveRecord::Base
+  self.table_name = 'pg_settings'
 
   virtual_belongs_to :vmdb_database
+  virtual_column :description,      :type => :string
+  virtual_column :minimum_value,    :type => :integer
+  virtual_column :maximum_value,    :type => :integer
+  virtual_column :vmdb_database_id, :type => :integer
 
-  attr_accessor :vmdb_database
+  attr_writer :vmdb_database
 
-  def initialize(values = {})
-    @vmdb_database = VmdbDatabase.my_database
-    super(values)
+  def vmdb_database
+    @vmdb_database ||= VmdbDatabase.my_database
```

Comment:
I only added the memoization because we have tests that call `vmdb_database=` and that sets the ivar for that test.  Maybe we could change the tests to remove the setting?